control-plane/plugin/Cargo.toml

```diff
@@ -36,6 +36,14 @@ serde_yaml = "0.8.26"
 humantime = "2.1.0"
 uuid = { version = "0.8.2", features = ["serde", "v4"] }
 serde_derive = "1.0.140"
+maplit = "1.0.2"
+k8s-openapi = { version = "0.15.0", features = ["v1_20"] }
```

Comment:
this is the rest plugin where all non-k8s specific commands go, there should be no k8s/kube usages here